engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/ConfluentUtils.scala

```diff
@@ -5,10 +5,12 @@ import java.nio.ByteBuffer
 import cats.data.Validated
 
 object ConfluentUtils {
+  //Copied from AbstractKafkaAvroSerDe.MAGIC_BYTE
+  final val magicByte = 0
```

Comment:
Regards to scala style guide, constants should be written like this: `MagicByte`.